conformance/lang/expressions/relational-expr/greater_than_expression.balt

```diff
@@ -0,0 +1,1297 @@
+Test-Case: output
+Description: Test greater than expression for positive integers.
+Labels: DecimalNumber, int, relational-expr
+
+function init() {
+   int a = 1;
+   int b = 101;
+   int c = 0;
+   int d = 9223372036854775807;
+   io:println(a > b); // @output false
+   io:println(c > a); // @output false
+   io:println(a > c); // @output true
+   io:println(d > a); // @output true
+   io:println(b > d); // @output false
+   io:println(a > a); // @output false
+}
+
```

Comment:
From the spec 
`
A relational-expr is evaluated by first evaluating the two expressions
`

Add a test with two function invocations as operands and check both functions being invoked

Reply:
Already have added test where both operands are invoking functions. 
https://github.com/rpjayasekara/ballerina-spec/blob/26a0700355d3e7ad16b36699c0f2fc595babbf07/conformance/lang/expressions/relational-expr/greater_than_expression.balt#L1095